Prep changelog and bump version string to final (#9770)
learningequality/kolibri: CHANGELOG.md

List of the most important changes for each release.

## 0.15.8

### Added

> * Add regression test and fix for multiple worker processes duplicating jobs.

Reviewer: ```suggestion
```

> * Adds job storage sanity check

Reviewer: ```suggestion
* Adds job storage sanity check to ensure that Kolibri will not fail to start if the asynchronous job storage is malformed
```

### Changed
* Logging: remove unused simple formatter, add asctime to color formatter

> * Resume content by last interaction

Reviewer: ```suggestion
* Order resume content display by last interaction
```

* Upgrade morango and lower default sync chunk size through CLI

> * Make learner ids unique when assigned from groups and adhoc groups.

Reviewer: ```suggestion
* Make learners only appear once in reports when assigned from both groups and individually to lessons and quizzes.
```

> * Improve import workflow for collections

Reviewer: ```suggestion
* Persist collection tokens when switching between individual and bulk import workflows for channels
```

### Fixed

> * CSV Endpoint permissions and error handling

Reviewer: I'd suggest adding the worker processes fix here.
```suggestion
* CSV Endpoint permissions and error handling
* Adds fix for multiple worker processes duplicating jobs.
```

* Adds translated string for user kind in the user table
* Check for an array's length to avoid breaking errors
* Fixes Version logic not handling non-tripartite version strings
* Filters out empty nodes, add safety to breaking code

> * Controls overlapping content

Reviewer: ```suggestion
* Prevent controls for the PDF renderer from overlapping content
```

> * Fix quiz completion notification regression

Reviewer: ```suggestion
* Fix quiz completion regression which caused the notification to contain the incorrect score
```
